maven-wrapper-distribution/src/resources/mvnw

```diff
@@ -118,7 +118,7 @@ if [ -z "$JAVACMD" ] ; then
       JAVACMD="$JAVA_HOME/bin/java"
     fi
   else
-    JAVACMD="`\\unset -f command; \\command -v java`"
+    JAVACMD="$(\\unset -f command; \\command -v java)"
```

Comment:
This is probably a bug. Double  backslash will not work with `$()`:

```
user@host ~ $ JAVACMD="$(\\unset -f command; \\command -v java)" 
zsh: command not found: \unset
zsh: command not found: \command

user@host ~ $ JAVACMD=$(\\unset -f command; \\command -v java)  
zsh: command not found: \unset
zsh: command not found: \command

user@host ~ $ JAVACMD="$(\unset -f command; \command -v java)" 
unset: no such hash table element: command

user@host ~ $ bash -li
[user@host ~]$ JAVACMD="$(\unset -f command; \command -v java)"
[user@host ~]$ JAVACMD="$(\\unset -f command; \\command -v java)"
bash: \unset: Kommando nicht gefunden.
bash: \command: Kommando nicht gefunden.
[user@host ~]$ JAVACMD=`\\unset -f command; \\command -v java`
[user@host ~]$ 
```

Reply:
confirm on `GNU bash, version 5.2.15(1)-release (x86_64-apple-darwin21.6.0)`  this is ok:

```bash
JAVACMD="$(unset -f command; command -v java)"; echo $JAVACMD
```

Reply:
A backslash usually means "don't use the alias if it exists". The double backslash was probably used b/c of backticks which we got rid of.

Reply:
```
JAVACMD="$("unset" -f command; "command" -v java)"
```
This above seems to work on bash and zsh